code/validation/round-forecasts.R

```diff
@@ -0,0 +1,12 @@
+# Round all forecasts in data-processed
+library(here)
+library(purrr)
+
+# Load forecast from individual file, round values, and re-write
+all_files <- dir(here("data-processed"), recursive = TRUE,
```

Comment:
It would be slightly more robust to use `dir(..., full.names = TRUE)` so you don't have to re-use `here("data-processed")` after.

This way, if we ever change the location / name of the folder containing the forecast, we just have to edit one line.

It will return absolute file paths but we don't really care since it's just an intermediary step that is never saved or displayed :shrug:

Reply:
I think you can drop the `here()` entirely in `read/write_csv()` :wink: